### PR TITLE
Apply all parts I have introduced to the existing plugins

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by tboerger** [comment]
@bradrydzewski I have applied all comments from the streamline PRs.

**Comment by kzaitsev** [comment]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+      branch: master
       event: push
 
-publish:
-  docker:
-    username: $$DOCKER_USER
-    password: $$DOCKER_PASS
-    email: $$DOCKER_EMAIL
+  develop:
+    image: docker
```
Comment (bradrydzewski) [inline .drone.yml]: this should be removed since we are now using tags (ie `1.0.0`) for versioning

**Comment by tboerger** [comment]
@bradrydzewski since the 0.5 plugins are now using mostly govendor I have also updated the readme now.

**Comment by tboerger** [comment]
@bradrydzewski what about this? I will close that for now but feel free to reopen it.

**Pull request closed without merging**